<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AOSC_Installer_MainWindow</class>
 <widget class="QMainWindow" name="AOSC_Installer_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AOSC_Installer_MainWindow</string>
  </property>
  <property name="whatsThis">
   <string extracomment="安同系统安装器"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="GetStartLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>准备开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="ReadingLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>阅读协议</string>
    </property>
   </widget>
   <widget class="QLabel" name="PartedLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>分区磁盘</string>
    </property>
   </widget>
   <widget class="QLabel" name="InstallSystemLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>建立系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="SetUserLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设置用户</string>
    </property>
   </widget>
   <widget class="QLabel" name="AllDoneLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>完成安装</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>安装步骤</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PervStepButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>550</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>后退</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextStepButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>550</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>前进</string>
    </property>
   </widget>
   <widget class="QFrame" name="CenterProcess">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>40</y>
      <width>600</width>
      <height>505</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
